src/app/auth/auth.js

```diff
@@ -191,4 +126,6 @@ $(document).ready((function ($) {
         }
     });
 
```

Comment:
`$('.alert .close').click(function () {` can be used globally, move it to `app.js` file

```javascript
    $('.alert .close').click(function () {
        var $alert = $(this).parent();
        $alert.fadeOut();
        $alert.dequeue();
    });
```

Reply:
Done

Reply:
Resolved.